src/main/java/org/alliancegenome/curation_api/jobs/executors/GeneExpressionExecutor.java

```diff
@@ -37,23 +45,61 @@ public void execLoad(BulkLoadFileHistory bulkLoadFileHistory) {
 			}
 			bulkLoadFileDAO.merge(bulkLoadFileHistory.getBulkLoadFile());
 
-			bulkLoadFileHistory.setCount(geneExpressionIngestFmsDTO.getData().size());
+			bulkLoadFileHistory.setCount(ANNOTATIONS, geneExpressionIngestFmsDTO.getData().size());
 			updateHistory(bulkLoadFileHistory);
-			
+
 			List<Long> annotationIdsLoaded = new ArrayList<>();
 			List<Long> annotationIdsBefore = geneExpressionAnnotationService.getAnnotationIdsByDataProvider(dataProvider);
 
+			List<Long> experimentIdsLoaded = new ArrayList<>();
+			List<Long> experimentIdsBefore = geneExpressionExperimentService.getExperimentIdsByDataProvider(dataProvider);
+
+			bulkLoadFileHistory.setCount(ANNOTATIONS, geneExpressionIngestFmsDTO.getData().size());
 			boolean success = runLoad(geneExpressionAnnotationService, bulkLoadFileHistory, dataProvider, geneExpressionIngestFmsDTO.getData(), annotationIdsLoaded);
```

Comment:
The generic runLoad method is going to count "Records" as it just has methods like `history.incrementCompleted()` rather than `history.incrementCompleted(countType)`.  This should be refactored to allow passing in an additional parameter to specify the name of the count (ANNOTATIONS in this case), but with a wrapper method to keep the default behaviour if no parameter passed in.

Reply:
runLoad refactored